pgsqltoolsservice/driver/driver_manager.py

```diff
@@ -0,0 +1,34 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from abc import ABC, abstractmethod
+from pgsqltoolsservice.parsers.owner_uri_parser import get_attribute_value
+from pgsqltoolsservice.driver.types import *
+
+class DriverManager:
+    """Wrapper class that handles different types of drivers and connections """
+
+    def __init__(self, provider, params):
+
+        # Get info about this connection's provider
+        self._provider = provider
+
+        # Create a connection using the provider and connection options
+        self._conn_object = self._create_connection(params.connection.options)
+
+    def _create_connection(self, options) -> ServerConnection:
+        """
+        Creates a ServerConnection according to the provider and connection options
+        :param options: a dict containing connection parameters
+        """
+        if self._provider == "PGSQL":
```

Comment:
Lets move the provider names to constants if not already present

Reply:
Done!